akka-http-caching/src/main/resources/reference.conf

```diff
@@ -0,0 +1,15 @@
+akka.http.caching {
+
+  # default configuration values for LfuCache
+  lfu-cache {
+
+    max-capacity = 512
+
+    initial-capacity = 16
+
```

Comment:
```
# Upper limit to the time period an entry is allowed to remain in the cache.
# Set to 'infinite' to disable eviction based on time.
```

Reply:
Caffeine's `expireAfterWrite` is based on write time (create or update). An `Expiry` can be used to set more advanced policies, such as create only.